src/font.rs

```diff
@@ -263,7 +267,7 @@ impl<'a> EncodingFormat4<'a> {
         let id_delta_pos = Self::get_id_deltas_position(seg_count) + seg_index_pos;
         let id_delta = get_i16(data, id_delta_pos as usize).unwrap();
         if id_range_offset_value == 0 {
-            Some((code_point as i16 + id_delta) as u16)
+            Some(code_point.wrapping_add(wrapping_i16_as_u16(id_delta)))
```

Comment:
Would `code_point.wrapping_add(id_delta as u16)` work? The transmute shouldn't be necessary, `as` does the the conversion for ints. But I think the idea is right.

Reply:
The problem is that `id_delta` seems to be negative, which makes the `as` conversion wrap and panic in debug mode.

Reply:
Maybe do another wrapping_add of i16::MAX and then convert with `as`?